webplugin/js/app/kommunicate-client.js

```diff
@@ -179,4 +179,26 @@ Kommunicate.client = {
             },
         });
     },
+
+    /**get the entire chatListDetails by groupId 
+     * @param {Object} options 
+     * @param {Number} options.groupId
+     * @param {function} callback 
+     */
+    getChatListByGroupId: function (options, callback) {
+        var { groupId } = options;
```

Comment:
Will this work in IE?

Reply:
Also, what will happen if groupId is undefined?

Reply:
No this wouldn't have worked in IE. Changed it. If groupId is undefined then the api will return an error.